### PR TITLE
Feat: function execution timer

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

Adds a timer to in-progress function executions

## Test Plan

Manual

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {
     "extends": "./.svelte-kit/tsconfig.json",
     "compilerOptions": {
-        "lib": ["webworker"],
+        "lib": [
+            "webworker",
+            "DOM",
+            "ESNext"
```
Comment (TorstenDittmann) [inline tsconfig.json:7]: Do we even need those? I am pretty sure they are present in `"extends": "./.svelte-kit/tsconfig.json"`

**Review by TorstenDittmann — COMMENTED** [review]
There are some linting issues 👍🏻

File: tsconfig.json (hunk 1 above)
Comment (TGlide) [reply tsconfig.json:7]: They are, but by defining lib as webworker only, we're overriding svelte-kit's tsconfig's lib option, so we need to re-include it.

Comment (TorstenDittmann) [reply tsconfig.json]: ah okay, thought its actually extending :-D

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**